package-scripts.js

```diff
@@ -31,9 +31,9 @@ module.exports = {
     // Build.
     // - Libraries
     "build:lib:esm":
-      "cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js --source-maps",
+      "cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js",
     "build:lib:cjs":
-      "cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js --source-maps",
+      "cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.build.js --extensions .tsx,.ts,.jsx,.js",
```

Comment:
One problem with this PR, potentially, is that I don’t think our `wireit` cache will be invalidated by this change.  I might need to add this file to the wireit inputs.